doc/analysis-plan.md

```diff
@@ -0,0 +1,54 @@
+# Possible analyses
+### I. How to answer research questions 
+For variable community composition, we plan to plot them with PCA and spatial autocorrelation, identify dominant genus (simpson's index), abundance/importance (if have over 10 genus).
+
+	Dependent variables: community composition for each site, species richness, species evenness. 
+	Independent variables includes: pH, seasonality, sites (other environmental factor might be added). 
+
+We predict that the independent variables will affect the dependent variables, and conservation action will be dependent on the community composition, species richness and speices evenness of the sites. 
```

Comment:
Since our project focuses on the link between environmental factors and our dependent variables, I think maybe the conservation part doesn't need to be in this part of the document.

Reply:
yes I agree. I think the conservation part should be left out.